src/Resources/Custom TypeEditor Template.md

```diff
@@ -0,0 +1,314 @@
+Custom TypeEditor Template
+
+Since .NET 3.1 started to support the WinForms Runtime, a new WinForms designer
+was needed to support .NET applications. The work required a near-complete
+rearchitecting of the designer, as we responded to the differences between .NET
+and the .NET Framework based WinForms designer everyone knows and loves.
```

Comment:
I read and re-read this sentence but I'm still having a hard time parsing it, I suggest a re-write/word.
"_The work required a near-complete rearchitecting of the designer, as we responded to the differences between .NET and the .NET Framework based WinForms designer everyone knows and loves._"